backend/src/main/java/kr/touroot/travelplan/service/TravelPlanService.java

```diff
@@ -75,7 +80,20 @@ private void createPlanPlace(List<PlanPlaceCreateRequest> request, TravelPlanDay
         for (int order = 0; order < request.size(); order++) {
             PlanPlaceCreateRequest planRequest = request.get(order);
             Place place = getPlace(planRequest);
-            travelPlanPlaceRepository.save(planRequest.toPlanPlace(order, travelPlanDay, place));
+            TravelPlanPlace travelPlanPlace = travelPlanPlaceRepository.save(
+                    planRequest.toPlanPlace(order, travelPlanDay, place)
+            );
```

Comment:
inline으로 처리하기 보단 전 보통 분리하는 편인데 어떻게 생각하시나요?
```suggestion
            TravelPlanPlace planPlace = planRequest.toPlanPlace(order, travelPlanDay, place);
            TravelPlanPlace savedPlanPlace = travelPlanPlaceRepository.save(planPlace);
```

지금 방식은 뭔가 indent가 늘어난 것 같은 기분이라 조금 꺼려지네요..

Reply:
합리적이라고 생각해서 반영했습니다 👍